### ISSUE TITLE
Could you revisit the idea of making a default folder for unnamed configurations?

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#203]
Hi again. Just wanted to ask if it would be possible "from the factory" to automatically create a "Default" folder in the "...\shared\ptxprint\" folder to hold all the files needed by the PTXprint app if the user does NOT save a configuration?

Why? Because it is messy not to. 
By that I mean if you look at a working "shared\ptxprint" folder you will see something like this: 

![image](https://user-images.githubusercontent.com/65363206/91864668-7d262200-ec3e-11ea-9166-8b1d939e5626.png)

Granted, normal users won't have so many configurations, but I'm pretty sure they will end up with several variations as they try things. But the confusion here is, what can I safely delete, and what would be a bummer to delete?

![image](https://user-images.githubusercontent.com/65363206/91864861-c1b1bd80-ec3e-11ea-9d31-6259b0fc4290.png)

As best as I can tell, these are the files/folders that are the "Default" or "Unnamed" configuration files used by PTXprint if a user runs the software without saving anything. (I see where some of my config folders have an AdjLists folder too. I'm not sure what that is, but I suppose that folder might appear in this "root" folder too, if the user specifies adjustments but doesn't save the configuration as a named configuration, right?)

It would be so much neater if these files were put in a "default" folder, where PTXprint would create and maintain these files and folders for those "unnamed" cases. (The folder for this could be called anything, I'm just suggesting "default" for lack of a better label.)

Organized this way, then the only thing found in that "shared\ptxprint" folder would be a list of all of PTXprint's configurations. And then a user could safely delete any folder they don't need anymore. (And hopefully most users would not delete a "default" folder). 

Just a request. Hoping to keep things neat and all on the same level, rather than all mixed together. (And hackers would then know where to copy their "good" configuration files when they have a great default setup.)

**Comment by markpenny** [comment]
This has been done now and should be in the next release (1.1.x, or 1.2). We're also still waiting for the SIL signed certificate to be recognized by Microsoft so that the installation experience isn't so terrifying :-)

**Comment by davidc86** [comment]
This appears to work now. Thanks!! It certainly cleans up the main folder nicely. And I think it will help new users more easily figure out what is going on in there. I haven't tested this extensively, but I noticed when I installed v1.1.8 it moved most of my "default" stuff from the shared\ptxprint folder, though I still see a "NestedStyles.sty" file (which I didn't make and I don't know what it is) still in the ptxprint folder rather than in the Default folder. But maybe you aren't using that file anymore? I noticed it show up at one point in some of my configurations, but not in others, so I'm not sure what it does.
